src/components/alert/index.js

```diff
@@ -1,17 +1,15 @@
 import React from 'react';
 import SweetAlert from 'sweetalert2-react';
 
-function CustomAlert({
+const CustomAlert = ({
   title = '', text = '', icon = 'info', state, setState,
-}) {
-  return (
-    <SweetAlert
-      show={state}
-      title={title}
-      text={text}
-      icon={icon}
-      onConfirm={() => setState(!state)}
-    />
-  );
-}
+}) => {
+  <SweetAlert
+    show={state}
+    title={title}
+    text={text}
+    icon={icon}
+    onConfirm={() => setState(!state)}
+  />;
+};
 export default CustomAlert;
```

Comment:
files with jsx syntax must be named with .jsx 
[](https://www.educative.io/answers/how-to-rename-or-move-files-in-git)

Reply:
done